analysis/cQED_dataAnalysis/analyzeProcessTomo.m

```diff
@@ -13,8 +14,23 @@
 
 if(isempty(varargin))
```

Comment:
@dieris I know this style exists elsewhere in QLab but it's ugly there too. Matlab's keyword argument support is lousy but they do have an [inputParser](http://www.mathworks.com/help/matlab/ref/inputparser-class.html) that makes this much cleaner.  The  examples are another case where  Mathworks is moving to a more dispatch style.